src/mmw/js/src/analyze/models.js

```diff
@@ -54,6 +54,33 @@ var AnalyzeTaskModel = coreModels.TaskModel.extend({
     }
 });
 
+var AnalyzeTaskCollection = Backbone.Collection.extend({
+    model: AnalyzeTaskModel
+});
+
+AnalyzeTaskCollection.getSingleton = function(App, aoi) {
+    if (!App.analyzeCollection) {
```

Comment:
This might work correctly, but I was somewhat surprised that this only checks that the `analyzeCollection` doesn't exist. It seems like it should also check to see if `aoi` is equal to the `aoi` for the current `analyzeCollection` if it exists.